library/src/main/java/io/andref/rx/widgets/ListViewCardAdapter.java

```diff
@@ -0,0 +1,264 @@
+    {
+        mCellHeight = cellHeight;
+        mAvatarAlpha = avatarAlpha;
+        mIconAlpha = iconAlpha;
+        mAvatarTint = avatarTint;
+    }
+
+    @Override
+    public RecyclerView.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType)
+    {
+        switch (viewType)
+        {
+            default:
+                View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.rxw_avatar_with_two_lines_and_icon, parent, false);
+                final ViewHolder viewHolder = new ViewHolder(view, mCellHeight);
+
+                RxView.clicks(view)
+                        .takeUntil(RxView.detaches(parent))
+                        .subscribe(new Action1<Void>()
+                        {
+                            @Override
+                            public void call(Void aVoid)
+                            {
+                                mItemClicks.onNext(new Pair<>(mItems.get(viewHolder.getAdapterPosition()), viewHolder.getAdapterPosition()));
+                            }
+                        });
+
+                View iconView = view.findViewById(R.id.image_button);
+                RxView.clicks(iconView)
+                        .takeUntil(RxView.detaches(parent))
+                        .subscribe(new Action1<Void>()
+                        {
+                            @Override
+                            public void call(Void aVoid)
+                            {
+                                mIconClicks.onNext(new Pair<>(mItems.get(viewHolder.getAdapterPosition()), viewHolder.getAdapterPosition()));
+                            }
+                        });
+
+                return viewHolder;
+        }
+    }
+
+    @Override
+    public void onBindViewHolder(RecyclerView.ViewHolder holder, int adapterPosition)
+    {
+        switch (getItemViewType(adapterPosition))
+        {
+            default:
+                final ViewHolder viewHolder = (ViewHolder) holder;
+                viewHolder.bindItem(mItems.get(adapterPosition), mAvatarAlpha, mAvatarTint, mIconAlpha);
+                if (adapterPosition == getItemCount() - 1)
+                {
+                    if (mShowLastDivider)
+                    {
+                        viewHolder.showDivider();
+                    }
+                    else
+                    {
+                        viewHolder.hideDivider();
+                    }
+                }
+                break;
+        }
+    }
+
+    @Override
+    public int getItemCount()
+    {
+        return mItems != null ? mItems.size() : 0;
+    }
+
+    public Observable<Pair<ListViewCard.Item, Integer>> itemClicks()
+    {
+        return mItemClicks;
+    }
+
+    public Observable<Pair<ListViewCard.Item, Integer>> iconClicks()
+    {
+        return mIconClicks;
+    }
+
+    public void setItems(List<ListViewCard.Item> items)
+    {
+        mItems = items;
+        notifyDataSetChanged();
+    }
+
+    public List<ListViewCard.Item> getItems()
```

Comment:
Do we expose this in the parent view? I think that was removed. We should add it back.